sample-queries/sample-queries.json

```diff
@@ -1699,6 +1699,16 @@
         "tip": "Go to Modify Permissions tab and consent to User.Read permission to run this query",
         "skipTest": false
     },
+    {
+        "id": "83670efe-06f3-4777-bef9-00b1ef450985",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "get ediscovery case",
+        "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/ediscoverycase-get?view=graph-rest-beta",
+        "tip": "Replace {caseId} with the ID from an eDiscovery Case (https://graph.microsoft.com/beta/compliance/ediscovery/cases).",
```

Comment:
Nitpick: In keeping up with the standards of the other `tip` message formats across the document, I would suggest we write this line as: 
`This query requires a case id. To find the id of a case; you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases`

Reply:
Fixed